apps/public-docsite-v9/package.json

```diff
@@ -22,6 +22,8 @@
   },
   "dependencies": {
     "@fluentui/react": "^8.98.0",
+    "@fluentui/react-northstar": "^0.64.0",
```

Comment:
So now everything gets build again? 🥹 (any change in any v9 package will trigger v8/v9 build)